jdaviz/configs/cubeviz/plugins/tests/test_data_selection.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+
+@pytest.mark.filterwarnings('ignore:No observer defined on WCS')
+def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
+    app = cubeviz_helper.app
+    app.add_data(spectrum1d_cube, 'test[FLUX]')
+    app.add_data(spectrum1d_cube, 'test[IVAR]')
+    app.add_data_to_viewer('flux-viewer', 'test[FLUX]')
```

Comment:
Does the behavior change if you do this instead? Because I found out that the calls are not equivalent between `app.add_data` and `viz.load_data` in https://github.com/spacetelescope/jdaviz/pull/1315#pullrequestreview-974575870

```suggestion
    cubeviz.load_data(spectrum1d_cube, 'test[FLUX]')
    cubeviz.load_data(spectrum1d_cube, 'test[IVAR]', show_in_viewer=False)
```

Reply:
p.s. What is your actual intent with `app.add_data(spectrum1d_cube, 'test[IVAR]')`? It looks like you are still adding the same cube but just pretending it is IVAR even though it is not really? Did I misunderstand something here?

Reply:
Ok, I'll rename them to "cube1" and "cube2" and leave a note in the test to that effect.  The test currently only covers ensuring that checking/unchecking adds/removes entries from the viewer as expected, so doesn't care about linking at all.  If we ever extend the test to cover more advanced use-cases of scrubbing multiple cubes, etc, then we will should also change it to use data that will test the linking logic.